charts/histogram/src/chart-settings/chart-settings.js

```diff
@@ -67,6 +67,12 @@ function createChartSettings(chartView, layout) {
     theme,
     isRtl,
   });
+  const themeService = createThemeService({
+    theme,
+    config: {
+      id: chartID,
```

Comment:
This should be `histogram` and not `object.histogram`.

The theme service uses the id as the base path, e.g. `histogram.label.value.fontFamily`.
See https://help.qlik.com/en-US/sense-developer/May2023/Subsystems/Extensions/Content/Sense_Extensions/CustomThemes/custom-themes-properties.htm
https://github.com/qlik-trial/qlik-chart-modules/tree/main/src/services/theme-service#readme